full-service/src/db/txo.rs

```diff
@@ -635,6 +636,7 @@ pub trait TxoModel {
     ///| `max_spendable_value` | The upper limit for the spendable TxOut value to filter on |                                     |
     ///| `token_id`            | The id of a supported type of token to filter on           |                                     |
     ///| `default_token_fee`   | The default transaction fee in Mob network                 |                                     |
+    ///| `only_from_subaddress`| The subaddress from which spendable txos can be sourced    | (optional)                          |
```

Comment:
The main motivation of this PR: the ability to select spendable txos _only_ from a give subaddress